.github/workflows/main.yml

```diff
@@ -452,6 +452,10 @@ jobs:
         -DWITH_TESTS=ON ..
         make -j 2
 
+    - uses: actions/checkout@v2
+    - name: Setup tmate session
```

Comment:
Do we want to leave this action? I thought it's only for interactive debugging and shouldn't be included in automated tests?

Reply:
That's right. I will remove this.